packages/components/src/constants/keyboardEvents.ts

```diff
@@ -0,0 +1,29 @@
+export enum KEYBOARD_CODE {
+    ENTER = 'Enter',
+    ESCAPE = 'Escape',
+    SPACE = 'Space',
+    BACK_SPACE = 'Backspace',
+    ARROW_RIGHT = 'ArrowRight',
+    TAB = 'Tab',
+
+    DIGIT_ONE = 'Digit1',
```

Comment:
Maybe with numbers it would be more clear at first glance `DIGIT_1`, `NUMPAD_5`,... What do you think?

Reply:
I tried that and finally decided to go for text, since it seemed more readable to me.
But I do not have any hard preferences. I can change it if you wish it.